modules/manage/pages/kubernetes/upgrade-kubernetes.adoc

```diff
@@ -51,7 +51,7 @@ rpk topic alter-config [<topic-name>,] --set replication.factor=<replication-fac
 +
 After increasing the replication factor, xref:./monitor.adoc#under-replicated-partitions[monitor for under-replicated partitions] and wait until all partitions are replicated.
 
-* *Delete PVCs*: If you are using local-storage-backed PVs in Kubernetes version 1.26 or earlier, make sure to delete the PersistentVolumeClaims (PVC) before upgrading the worker node. Otherwise, your Pods will remain in a pending state when they are rescheduled because the PVs are bound to the node through node affinity. Keep in mind that deleting a PVC does not remove it from the system immediately. Instead, a `deletionTimestamp` is set on the PVC, but it will not be deleted until the associated Pod is terminated.
+* *Delete PVCs*: If you are using local PVs in Kubernetes version 1.26 or earlier, make sure to delete the PersistentVolumeClaims (PVC) before upgrading the worker node. Otherwise, your Pods will remain in a pending state when they are rescheduled because the PVs are bound to the node through node affinity. Keep in mind that deleting a PVC does not remove it from the system immediately. Instead, a `deletionTimestamp` is set on the PVC, but it will not be deleted until the associated Pod is terminated.
```

Comment:
*Delete PVCs*: If you are using local PVs in Kubernetes version 1.26 or earlier, be sure to delete the PersistentVolumeClaims (PVC) before upgrading the worker node. Otherwise, your Pods will remain in a pending state when they are rescheduled because the PVs are bound to the node through node affinity. Note that deleting a PVC does not remove it from the system immediately. Instead, a `deletionTimestamp` is set on the PVC, but it is not deleted until the associated Pod is terminated.